management/WFCORE-7235_Platform_MBeans_Refresh.adoc

```diff
@@ -0,0 +1,182 @@
+---
+categories:
+- management
+stability-level: default
+issue: https://github.com/wildfly/wildfly-proposals/issues/728
+feature-team:
+ developer: jfdenise
+ sme:
+  - bstansberry
+ outside-perspective:
+  -
+---
+= Platform MBean evolution
+:author:            Jean-Francois Denise
+:email:             jdenise@redhat.com
+:toc:               left
+:icons:             font
+:idprefix:
+:idseparator:       -
+
+== Overview
+
+== Issue Metadata
+
+=== Issue
+
+* https://issues.redhat.com/browse/WFCORE-7235[WFCORE-7235]
+
+=== Related Issues
+
+=== Stability Level
+// Choose the planned stability level for the proposed functionality
+* [ ] Experimental
+
+* [ ] Preview
+
+* [ ] Community
+
+* [x] default
```

Comment:
Given your feature team members this can't be default.

I think this should be segregated into two aspects -- bug fixing to bring what is exposed by the currently supported java.lang.management.XXXMXBean impls up to date, at default stability, and then the feature here is the logging MXBean and the com.sun.management stuff. Those 'feature' things should be at community stability.

I'm not suggesting you separate the work into different commits, file different issues etc, unless you want to. The 'bug fix' aspect could just be described in the requirements section.